Dockerfile

```diff
@@ -9,13 +15,6 @@ ENV REPO_PATH=/managed-cluster-config
 COPY . ${REPO_PATH}
 WORKDIR ${REPO_PATH}
 
-# Docker dependencies:
-RUN yum update -y && yum install -y \
-    git \
-    make \
-    epel-release
-RUN yum -y install python-pip && yum clean all
-
 # Upgrade pip and install necessasry packages
 RUN pip install \
```

Comment:
Since pip is already installed in the img, I assume we don't need to install/upgrade it in the line below